modules/integration/tests-common/admin-clients/src/main/java/org/wso2/identity/integration/common/clients/application/mgt/DefaultAuthSeqMgtServiceClient.java

```diff
@@ -0,0 +1,195 @@
+
+        try {
+            if (debugEnabled) {
+                log.debug("Retrieving default authentication sequence.");
+            }
+            return stub.getDefaultAuthenticationSeqInXML(DEFAULT_AUTH_SEQ);
+        } catch (RemoteException e) {
+            log.error("Error occurred when retrieving default authentication sequence.", e);
+            throw new IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException("Server error occurred.");
+        }
+    }
+
+    /**
+     * Retrieve default authentication sequence.
+     *
+     * @return default authentication sequence
+     * @throws IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException
+     */
+    public DefaultAuthenticationSequence getDefaultAuthenticationSeq() throws
+            IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException {
+
+        try {
+            if (debugEnabled) {
+                log.debug("Retrieving default authentication sequence.");
+            }
+            return stub.getDefaultAuthenticationSeq(DEFAULT_AUTH_SEQ);
+        } catch (RemoteException e) {
+            log.error("Error occurred when retrieving default authentication sequence.", e);
+            throw new IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException("Server error occurred.");
+        }
+    }
+
+    /**
+     * Retrieve default authentication sequence basic info.
+     *
+     * @return default authentication sequence
+     * @throws IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException
+     */
+    public DefaultAuthenticationSequence getDefaultAuthenticationSeqInfo()
+            throws IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException {
+
+        try {
+            if (debugEnabled) {
+                log.debug("Retrieving default authentication sequence info.");
+            }
+            return stub.getDefaultAuthenticationSeqInfo(DEFAULT_AUTH_SEQ);
+        } catch (RemoteException e) {
+            log.error("Error occurred when retrieving default authentication sequence info.", e);
+            throw new IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException("Server error occurred.");
+        }
+    }
+
+    /**
+     * Check existence of default authentication sequence.
+     *
+     * @return true if a default authentication sequence exists for the tenant
+     * @throws IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException
+     */
+    public boolean isExistingDefaultAuthenticationSequence()
+            throws IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException {
+
+        try {
+            if (debugEnabled) {
+                log.debug("Checking existence of default authentication sequence.");
+            }
+            return stub.isExistingDefaultAuthenticationSequence(DEFAULT_AUTH_SEQ);
+        } catch (RemoteException e) {
+            log.error("Error while check existence of default authentication sequence.", e);
```

Comment:
This should be "Error while checking existence of default authentication sequence."